fineract-provider/src/main/java/org/apache/fineract/portfolio/shareproducts/service/ShareProductCommandsServiceImpl.java

```diff
@@ -47,7 +47,7 @@ public CommandProcessingResult postDividends(Long productId, JsonCommand jsonCom
     public Object handleCommand(Long productId, String command, String jsonBody) {
         final JsonElement parsedCommand = this.fromApiJsonHelper.parse(jsonBody);
         final JsonCommand jsonCommand = JsonCommand.from(jsonBody, parsedCommand, this.fromApiJsonHelper, null, null, null, null, null,
-                null, null, null, null, null);
+                null, null, null, null, null,null);
```

Comment:
from static method of JsonCommand class has to be chaned in order to get status value from the JSONCommand.  The change in the signature results in addition of one more parameter value in this class.